core/src/main/scala/org/apache/spark/storage/memory/MemoryStore.scala

```diff
@@ -252,7 +250,7 @@ private[spark] class MemoryStore(
         if (unrollMemoryUsedByThisBlock <= size) {
```

Comment:
Here the size of underlying vector or bytebuffer maybe greater than the `unrollMemoryUsedByThisBlock `.